src/components/sbb-form-field/sbb-form-field.tsx

```diff
@@ -263,7 +263,11 @@ export class SbbFormField implements ComponentInterface {
     return (
       <div class="sbb-form-field__space-wrapper">
         {/* eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions */}
-        <div onClick={(event) => this._handleWrapperClick(event)} class="sbb-form-field__wrapper">
+        <div
+          onClick={(event) => this._handleWrapperClick(event)}
+          class="sbb-form-field__wrapper"
+          id="form-field-wrapper"
```

Comment:
Why is the id necessary?

Reply:
As @jeripeierSBB said.
I added the id because I already felt dirty querying an element inside the shadow dom of another component and didn't want to create a dependency on the class that could change for other reasons.

I'll add a comment beside di id so that we know why it is necessary